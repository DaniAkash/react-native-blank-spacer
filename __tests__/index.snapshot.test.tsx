import React from "react";
import renderer from "react-test-renderer";
import BlankSpacer from "../src/index";

it("renders correctly", () => {
  const tree = renderer
    .create(
      <BlankSpacer
        height={20}
        width={30}
        color={"green"}
        style={{ margin: 24, padding: 24 }}
      />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
