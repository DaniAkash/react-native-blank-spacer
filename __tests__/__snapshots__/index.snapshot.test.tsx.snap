// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "green",
        "height": 20,
        "width": 30,
      },
      Object {
        "margin": 24,
        "padding": 24,
      },
    ]
  }
/>
`;
